packages/lms/src/lib/components/ui/Icon.svelte

```diff
@@ -1,6 +1,8 @@
 <script lang="ts">
   export let name: string;
+  export let variant:string = "";
```

Comment:
```suggestion
  export let variant = "";
```

Type is inferred from the value assignment.